hat/extract_timeseries/extractor.py

```diff
@@ -1,128 +1,210 @@
-from dask.diagnostics import ProgressBar
+from dask.diagnostics.progress import ProgressBar
```

Comment:
Is this coming from a specific version of dask? should we add that as a requirement in `pyproject.toml` ?

Reply:
Ah, I think this can be reverted. I just changed it because my linter complained and proposed this more specific import.

```suggestion
from dask.diagnostics import ProgressBar
```